src/tests/Interop/Swift/README.md

```diff
@@ -0,0 +1,96 @@
+# .NET Swift Interop
+
+The Swift programming language has a different ABI, runtime environment, and object model, making it challenging to call from the .NET runtime. Existing solutions, like [Binding Tools for Swift](https://github.com/xamarin/binding-tools-for-swift) and [BeyondNet](https://github.com/royalapplications/beyondnet).
+
+This project aims to explore the possibilities and limitations of direct P/Invoke interop with Swift. It should implement runtime mechanisms for handling Swift ABI differences. For a comprehensive .NET Swift interop, the Binding Tools for Swift contains valuable components that could be reused or built upon.
+
+We want to focus on the runtime support for direct P/Invoke interop with Swift, which Xamarin bindings will consume to support running Maui apps with third-party Swift libraries. Ideally, we want to avoid generating extra wrappers and attempt to directly call all kinds of Swift functions. While external tools can be helpful, they may introduce additional complexity, and we intend to integrate them into the .NET toolchain ecosystem.
+
+## Swift ABI in a nutshell
```

Comment:
This is great information. I think this should be somewhere similar to where we have our Objective-C support. For example https://github.com/dotnet/designs/blob/b70a81ac399e0737be73ff3acfb8a0591ff395e9/accepted/2021/objectivec-interop.md captured our Objective-C details. I think having something under https://github.com/dotnet/runtime/tree/main/docs/design

Reply:
Do you think this is needed at all? This seems to overlap with the design document and may be redundant.

Reply:
Nope. Removal and deference to the design doc make sense to me.

Reply:
Removed.